src/mlpack/methods/ann/activation_functions/elish_function.hpp

```diff
@@ -0,0 +1,122 @@
+/**
+ * @file elish_function.hpp
+ * @author Bisakh Mondal
+ *
+ * Definition and implementation of the ELiSH function as described by
+ * Mina Basirat and Peter M. Roth.
+ *
+ * For more information see the following paper
+ * 
+ * @code
+ * @misc{ELiSHActivation,
+ *    title={The Quest for the Golden Activation Function},
+ *    author={Mina Basirat and Peter M. Roth},
+ *    year={2018},
+ *    url={https://arxiv.org/pdf/1808.00783.pdf},
+ *    eprint={1808.00783},
+ *    archivePrefix={arXiv},
+ *    primaryClass={cs.NE} }
```

Comment:
Hey @bisakhmondal, Could you add space between operator (`=`). Thanks.

Reply:
sure.